modules/backend/formwidgets/Relation.php

```diff
@@ -123,6 +123,18 @@ protected function makeRenderFormField()
             // by joining its pivot table. Remove all joins from the query.
             $query->getQuery()->getQuery()->joins = [];
 
+            // Remove sorting on columns of joined tables including pivots
+            $pivotColumns = $this->model->$relationType[$relationObject->getRelationName()]['pivot'] ?? [];
+
+            $query->getQuery()->getQuery()->orders = array_filter(
+                $query->getQuery()->getQuery()->orders,
+                function ($order) use ($relationModel, $pivotColumns) {
+                    // keep table own columns and ignore pivot columns
+                    return explode('.', $order['column'])[0] === $relationModel->getTable() ||
+                    !in_array($order['column'], $pivotColumns);
```

Comment:
@motia Would you mind reformatting this `return` statement? It is a little hard to read the separation of conditions.
```php
                    $ownColumn = explode('.', $order['column'])[0] === $relationModel->getTable();
                    return $ownColumn || !in_array($order['column'], $pivotColumns);
```